modules/ballerina-test/src/test/resources/test-src/statements/foreach/foreach-complex.bal

```diff
@@ -45,7 +45,7 @@ function testIntRangeSimple(int a, int b)(string){
 
 function testIntRangeEmptySet()(string){
     output = "";
-    foreach i,j in 5 .. 0  {
+    foreach i,j in 5.. 0  {
```

Comment:
The spaces added/removed in this file are intentional, as they test different lexer conditions.